pkg/rancher-desktop/assets/translations/en-us.yaml

```
@@ -4742,6 +4742,15 @@ snapshots:
     icon: icon-search
     heading: No snapshots found
     body: Click on Create Snapshot to get started
+  info:
+    restore:
+      success: '{snapshot} was successfully restored'
```

I think we can be more concise with a few of our messages

```suggestion
      success: 'Restored {snapshot}'
```